influxql/parser.go

```diff
@@ -372,20 +374,20 @@ func (p *Parser) parseInt(min, max int) (int, error) {
 	return n, nil
 }
 
-// parseUInt32 parses a string and returns a 32-bit unsigned integer literal.
-func (p *Parser) parseUInt32() (uint32, error) {
+// parseUInt parses a string and returns a 64-bit unsigned integer.
+func (p *Parser) parseUInt(base int, bitSize int) (uint64, error) {
```

Comment:
I started to add a separate `parseUInt64` func but that's kind of a waste.  This function is really just a wrapper around the stdlib's `strconv.ParseUint` function that returns a parser error if needed.  I added the `base` param just to be consistent with the stdlib func it's wrapping.